LIMS2DB/classes.py

```diff
@@ -1079,6 +1079,10 @@ def set_status(self):
                         status_fields['status'] = 'Ongoing'
                         status_fields['ongoing'] = True
                         status_fields['open'] = True
+                    elif self.obj.get('project_summary', {}).get('queued'):
```

Comment:
Could be an an `or` with the condition above

Reply:
True! Done!